src/constants/categories.js

```diff
@@ -11,21 +11,57 @@ export default [
         name: 'Employment',
         tint: '040',
         shade: '600',
-        indicatorIds: [
-          'participationRate',
-          'gdp',
+        summaryIndicatorIds: [
```

Comment:
Changed the structure of this file so that summary indicators are decoupled from charts. Also, charts can render with one or more indicators.